class/lieutenant.yml

```diff
@@ -4,45 +4,45 @@ parameters:
       # CRDs
       - type: https
         source: ${lieutenant:operator:manifest_url}/crds/syn.tools_clusters_crd.yaml
-        output_path: manifests/crds/${lieutenant:operator:manifest_version}/syn.tools_clusters_crd.yaml
```

Comment:
If the old output paths would have been correctly prefixed with `dependencies/lieutenant`, the following change in the Makefile would be sufficient to fix the unit tests (cf. change to the component template in https://github.com/projectsyn/commodore/pull/349):

In `Makefile`:

```makefile
.PHONY: compile
compile: format lint
	@echo Compiling..
  mkdir -p dependencies
	@$(COMMODORE_CMD) component compile . -f tests/test.yml --search-paths ./dependencies
```

Reply:
I somehow thought there was a reason that this wouldn't work, but it seems to work fine now. You're right that's a lot better that way.